backend/src/main/java/gov/cdc/usds/simplereport/api/model/errors/TestEventSerializationFailureException.java

```diff
@@ -0,0 +1,32 @@
+package gov.cdc.usds.simplereport.api.model.errors;
+
+import graphql.ErrorClassification;
+import graphql.ErrorType;
+import graphql.GraphQLError;
+import graphql.language.SourceLocation;
+import java.util.List;
+import java.util.UUID;
+
+/**
+ * Exception to throw when a TestEvent fails to serialize when saving to the ReportStream uploader
+ * queue
+ */
+public class TestEventSerializationFailureException extends RuntimeException
+    implements GraphQLError {
```

Comment:
Commenting in case anyone else was wondering - this GraphQlError still returns a 200 to the frontend, I assume with success=false. I don't imagine this change will really increase the chances of throwing triggering the corresponding page since I assume the same thing happens today, just wanted to make a note of it.